packages/rich-text/src/plugins/Editor/index.ts

```diff
@@ -34,20 +34,28 @@ export function withEditorPlugin(editor: CustomEditor): CustomEditor {
     const LIST_TYPES: string[] = [BLOCKS.OL_LIST, BLOCKS.UL_LIST];
     const isActive = isBlockSelected(type);
     const isList = LIST_TYPES.includes(type);
+    const isQuote = type === BLOCKS.QUOTE;
 
     Transforms.unwrapNodes(editor, {
       match: (node) =>
         !Editor.isEditor(node) &&
         Element.isElement(node) &&
-        LIST_TYPES.includes((node as CustomElement).type),
+        (LIST_TYPES.includes((node as CustomElement).type) ||
+          BLOCKS.QUOTE.includes((node as CustomElement).type)),
       split: true,
     });
     const newProperties: Partial<CustomElement> = {
-      type: isActive ? BLOCKS.PARAGRAPH : isList ? BLOCKS.LIST_ITEM : type,
+      type: isActive
+        ? BLOCKS.PARAGRAPH
+        : isList
+        ? BLOCKS.LIST_ITEM
+        : isQuote
+        ? BLOCKS.PARAGRAPH
+        : type,
```

Comment:
Should we find something more readable for this case? Maybe we will need to add more conditions here and a ternary if will not be the best to read.